src/drivers/beanstalk/Command.php

```diff
@@ -60,7 +60,7 @@ public function actionRun()
      */
     public function actionListen()
     {
-        $this->queue->listen();
+        $this->queue->run(true, 3);
```

Comment:
What is 3?

Reply:
This is number of seconds to wait message from Beanstalk socket. It happened historically that scalar value is used. I will move the definition to console parameter.